flang/docs/FortranStandardsSupport.md

```diff
@@ -0,0 +1,119 @@
+<!--===- docs/FortranStandardsSupport.md
+
+   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+   See https://llvm.org/LICENSE.txt for license information.
+   SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+-->
+
+# Flang Fortran Standards Support
+
+```{contents}
+---
+local:
+---
+```
+
+This document summarizes Fortran standards support in Flang. The information is only provided as a guideline. The
+TODOs/Not Yet Implemented messages emitted by the compiler for unimplemented features should be treated as authoritative.
+
+The standards support information is provided as a table with three columns that are self explanatory. No distinction is
+made between the support in the Parser/Semantics and MLIR or Lowering support. The Status column uses
+the letters **P**, **Y**, **N** for the implementation status:
+- **P** : When the implementation is incomplete for a few cases
+- **Y** : When the implementation is complete
+- **N** : When the implementation is absent
```

Comment:
I still think we should make this easier for readers (explain the letter, order them by degree of completion)
```suggestion
the letters **Y**, **P**, **N** for the implementation status:
- **Y** (Yes): When the implementation is complete
- **P** (Partial): When the implementation is incomplete
- **N** (No): When the implementation is absent
```

Reply:
Done. The following format was used.

```
- **Y** : Yes. When the implementation is complete
```